<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorWindow</class>
 <widget class="QWidget" name="EditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">SQL editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBar" name="toolBar"/>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="query">
      <attribute name="title">
       <string>Query</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="SqlEditor" name="sqlEdit">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
         </widget>
         <widget class="QWidget" name="resultsContainer" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="results">
      <attribute name="title">
       <string notr="true">Results</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QWidget" name="resultsFrame" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="DataView" name="dataView">
            <property name="tabPosition">
             <enum>QTabWidget::South</enum>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="history">
      <attribute name="title">
       <string>History</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTableView" name="historyList">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
         <widget class="SqlView" name="historyContents">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SqlView</class>
   <extends>QPlainTextEdit</extends>
   <header>sqlview.h</header>
  </customwidget>
  <customwidget>
   <class>DataView</class>
   <extends>QTabWidget</extends>
   <header>dataview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SqlEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>sqleditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
